app/helpers/companies_helper.rb

```diff
@@ -4,9 +4,18 @@ def company_complete? company
     !company.nil? && !company.name.nil? && !company.name.empty? &&
       !company.region.nil? && !company.region.empty?
   end
+
+  def last_category_name company
+    if company.business_categories.any?
+      company.business_categories.last.name
+    end
+
+  end
```

Comment:
Would it be better to return "None" rather than `nil` since this is used in views?